src/web/views/locales/en/cookies.html

```diff
@@ -0,0 +1,87 @@
+    manage cookies on the Information Commissioners' website.</a></p>
+
+<h2 class="govuk-heading-m">Google Analytics cookie</h2>
+
+<p class="govuk-body">We use Google Analytics to collect information about how you use the service. This information
+    helps us to make improvements.</p>
+
+<p class="govuk-body">The Google analytics cookie collects and stores information about:</p>
+<ul class="govuk-list govuk-list--bullet">
+    <li>the pages you visit - how long you spend on each page</li>
+    <li>how you got to the service</li>
+    <li>what you click on while you are using the service</li>
+</ul>
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th class="govuk-table__header" scope="col">Name</th>
+            <th class="govuk-table__header" scope="col">Purpose</th>
+            <th class="govuk-table__header" scope="col">Expires</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_ga</td>
+            <td class="govuk-table__cell">Used to identify unique users and inform referring sites, visitor and session
+                counts.
+            </td>
+            <td class="govuk-table__cell">2 years</td>
+        </tr>
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gid</td>
+            <td class="govuk-table__cell">Used to identify unique users and inform referring sites, visitor and session
+                counts.
+            </td>
+            <td class="govuk-table__cell">24 hours</td>
+        </tr>
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gat_UA-133839203_4</td>
+            <td class="govuk-table__cell">Used to protect the Google Analytics server by limiting the number of requests
+                being made.
+            </td>
+            <td class="govuk-table__cell">1 minute</td>
+    </tr>
+    </tbody>
+</table>
+
+<p class="govuk-body">Google is not allowed to use or share our analytics data with anyone.</p>
+
+<h2 class="govuk-heading-m">Session cookies</h2>
+
+<p class="govuk-body">We store session cookies on your computer to remember your information and keep it secure while
+    you use the service.</p>
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th class="govuk-table__header" scope="col">Name</th>
+            <th class="govuk-table__header" scope="col">Purpose</th>
+            <th class="govuk-table__header" scope="col">Expires</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">htbhf.sid</td>
+            <td class="govuk-table__cell">Used for session management.</td>
+            <td class="govuk-table__cell">When you close your browser.</td>
```

Comment:
This will change, but we can update it in HTBHF-483

Reply:
Yes, for now I just copied the copy that I was given by Kate.